mne/decoding/time_gen.py

```diff
@@ -485,23 +484,26 @@ def score(self, epochs=None, y=None, scorer=None, test_times=None):
         else:
             transform_classes = False
 
-        # Initialize values: Note that this is not an array as the testing
-        # times per training time need not be regular
-        scores = [list() for _ in range(len(self.test_times_['slices']))]
-
-        # Loop across training/testing times
-        for t_train, slices in enumerate(self.test_times_['slices']):
-            n_time = len(slices)
-            # Loop across testing times
-            scores[t_train] = [0] * n_time
-            for t, indices in enumerate(slices):
-                y_true = self.y_true_
-                y_pred = self.y_pred_[t_train][t]
-                # Transform labels for Sklearn API compatibility
-                if transform_classes:
-                    y_true = le.transform(y_true)
-                # Scores across trials
-                scores[t_train][t] = _score(y_true, y_pred, scorer)
+        # Loop across estimators (i.e. training times)
+        y_true = self.y_true_
+        if transform_classes:
+            y_true = le.transform(y_true)
+
+        # Preprocessing for parallelization:
+        if self.n_jobs > 0:
+            n_chunk = self.n_jobs
+        else:
+            n_chunk = multiprocessing.cpu_count()
```

Comment:
I corrected this in the last commit, but sometimes, I need to get the n_jobs calling parallel_func because I need to bound it to the number of time sample / time chunk. Except if you have a better solution?